providers/oidc.go

```diff
@@ -185,24 +185,22 @@ func (p *OIDCProvider) CreateSessionStateFromBearerToken(ctx context.Context, ra
 	return newSession, nil
 }
 
-func (p *OIDCProvider) createSessionStateInternal(ctx context.Context, rawIDToken string, idToken *oidc.IDToken, token *oauth2.Token) (*sessions.SessionState, error) {
+func (p *OIDCProvider) createSessionStateInternal(ctx context.Context, idToken *oidc.IDToken, token *oauth2.Token) (*sessions.SessionState, error) {
```

Comment:
This is used by the normal flow as well right? We don't want to change the behaviour of the normal authentication flow. This would constitute a massive change in behaviour with this PR as is as far as I can tell.

Reply:
Let me check my logic again.  Everything should have the same logic flow/behavior (some of these strings just evaluated later in methods to remove redundant arguments) -- just refactored a bit as far as argument passing to these methods to fix some code climate code smell findings.